tests/rptest/scale_tests/high_throughput_test.py

```diff
@@ -0,0 +1,918 @@
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import itertools
+import math
+import re
+import time
+from typing import Optional
+
+from ducktape.mark import ok_to_fail
+from ducktape.tests.test import TestContext
+from ducktape.utils.util import wait_until
+from rptest.clients.rpk import RpkTool
+from rptest.services.cluster import cluster
+from rptest.services.failure_injector import FailureInjector, FailureSpec
+from rptest.services.kgo_verifier_services import (KgoVerifierProducer,
+                                                   KgoVerifierRandomConsumer)
+from rptest.services.metrics_check import MetricCheck
+from rptest.services.openmessaging_benchmark import OpenMessagingBenchmark
+from rptest.services.openmessaging_benchmark_configs import \
+    OMBSampleConfigurations
+from rptest.services.redpanda import (RESTART_LOG_ALLOW_LIST, MetricsEndpoint,
+                                      SISettings)
+from rptest.services.rpk_consumer import RpkConsumer
+from rptest.tests.prealloc_nodes import PreallocNodesTest
+from rptest.util import firewall_blocked
+from rptest.utils.node_operations import NodeDecommissionWaiter
+from rptest.utils.si_utils import nodes_report_cloud_segments
+
+KiB = 1024
+MiB = KiB * KiB
+GiB = MiB * KiB
+minutes = 60
+hours = 60 * minutes
+
+
+class CloudTierConfig:
+    def __init__(self, ingress_rate: float, egress_rate: float,
+                 num_brokers: int, num_brokers_scaled: int, segment_size: int,
+                 cloud_cache_size: int, partitions_min: int,
+                 partitions_max: int, connections_limit: Optional[int],
+                 rpo: int, memory_per_broker: int) -> None:
+        self.ingress_rate = int(ingress_rate)
+        self.egress_rate = int(egress_rate)
+        self.num_brokers = num_brokers
+        self.num_brokers_scaled = num_brokers_scaled
+        self.segment_size = segment_size
+        self.cloud_cache_size = cloud_cache_size
+        self.partitions_min = partitions_min
+        self.partitions_max = partitions_max
+        self.connections_limit = connections_limit
+        self.rpo = rpo
+        self.memory_per_broker = memory_per_broker
+        self.scaling_factor = num_brokers_scaled / num_brokers
+
+    @property
+    def segment_size_scaled(self):
+        return int(self.segment_size * self.scaling_factor)
+
+    @property
+    def partitions_max_scaled(self):
+        return int(self.partitions_max * self.scaling_factor)
+
+    @property
+    def ingress_rate_scaled(self):
+        return int(self.ingress_rate * self.scaling_factor)
+
+    @property
+    def egress_rate_scaled(self):
+        return int(self.egress_rate * self.scaling_factor)
+
+    @property
+    def connections_limit_scaled(self):
+        if self.connections_limit is None:
+            return None
+        return int(self.connections_limit * self.scaling_factor)
+
+
+# yapf: disable
+CloudTierConfigs = {
```

Comment:
```suggestion
ClusterConfigs = {
```
i suggest renaming this so it is not confused with existing cloud tier configuration settings that may drift away from what is defined in this file